frontend/src/components/dashboard/contactsHomePage/ContactSearch.js

```diff
@@ -51,6 +51,7 @@ export class ContactSearch extends Component {
                   ...this.props.partyRelationshipTypesList,
                 ]}
                 initialValues={this.props.initialValues}
+                enableReinitialize
```

Comment:
This should be added in the export like so: 
```
export default reduxForm({
  form: FORM.CONTACT_ADVANCED_SEARCH,
  touchOnBlur: false,
  enableReinitialize: true
})(AdvancedContactSearchForm);
```

Makes it easier to track what the form is enabling/disabling

Reply:
done